app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/base/BaseRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,25 @@
+package com.nguyennhatminh614.motobikedriverlicenseapp.utils.base
+
+import androidx.recyclerview.widget.AsyncDifferConfig
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.viewbinding.ViewBinding
+import com.nguyennhatminh614.motobikedriverlicenseapp.utils.OnClickItem
+import java.util.concurrent.Executors
+
+abstract class BaseRecyclerViewAdapter<T, VB : ViewBinding, VH : BaseViewHolder<T, VB>>(
```

Comment:
class base này em follow theo ở đâu thế, 
đính kèm link hộ anh, anh thấy nó Lạ lắm :D

Reply:
https://github.com/awesome-academy/The-meals/blob/develop/app/src/main/java/com/example/themeal/base/BaseAdapter.kt